about.html

```diff
@@ -0,0 +1,95 @@
+            <a href="">Join</a>
+            <a href="">Sponsor</a>
+        </nav>
+        <!--mobile navbar-->
+        <section class="mobile-nav">
+            <nav id="mobileSidebar" class="sidebar">
+                <a href="javascript:void(0)" class="closebtn" onclick="closeNav()">X</a>
+                <a href="#">About</a>
+                <a href="#">Program</a>
+                <a href="#">Join</a>
+                <a href="#">Sponsor</a>
+                <a href="./index.html">Home</a>
+            </nav>
+            <div id="main">
+                <button class="openbtn" onclick="openNav()">☰</button>
+            </div>
+        </section>
+        <section class="sectionOneAbout">
+            <div>
+                <p class="hello">"HELLO CONNECTED WORLD!"</p>
+                <h1>CONNECTIVITY IS UNLEASHED AT MWC BARCELONA 2022</h1>
+                <p class="sectionAbout">MWC Barcelona attracts some of the most influential decision makers in the
+                    world.
+                    It’s where 2,400 leading companies and trailblazers share the latest topics relevant to the future.
+                    And the best place for networking opportunities with mobile and tech industry influencers.</p>
+                <h4>Please contact us per Email for any further informations
+                    about Mobile World Congress 2021! <br>
+                    mwc2021@gmail
+                </h4>
+            </div>
+        </section>
+    </header>
+    <section class="secondSectionAbout">
+        <div class="logoSection">
+            <h2 class="logofont">The GSM Mobile World Congress 2021 logo is registered and owned by the GSM Association.</h2>
+            <img src="./img/logo2.png" alt="">
+        </div>
+    </section>
+    <section class="speackersPage">
+        <div class="container">
+            <div>
+                <h2 class="featured">See our previous MWC Congresses</h2>
+                <div class="line"></div>
+            </div>
+
+            <div class="about-img about-img-shangai">
+                <p class="imgtext">2015 MWC Congress Shangai</p>
+            </div>
+            <div class="about-img about-img-paris">
+                <p class="imgtext">2017 MWC Congress Los Angeles</p>
+            </div>
+        </div>
+    </section>
+    <section class="partnersection">
+        <h2 class="partnertext">Partners</h2>
+        <div class="partners">
+            <img class="partnericon" src="./img/sponsor1.jpg" alt="">
+            <img class="partnericon" src="./img/logospnsor2.png" alt="">
+            <img class="partnericon" src="./img/logosponsor4.png" alt="">
+            <img class="partnericon" src="./img/logosponsor5.png" alt="">
+            <img class="partnericon" src="./img/logosponsor6.png" alt="">
+        </div>
+    </section>
+    <footer>
+        <img src="./img/logo3.png" alt="">
+        <p>GSMA logoLegal© Copyright 2021 GSM Association.</p>
+    </footer>
```

Comment:
- [ ] The footer on desktop view has a dark background and a white background on mobile view. Please kindly fix that on both pages